templates/ipsec.conf

```diff
@@ -2,28 +2,50 @@ version 2.0
 
 config setup
   dumpdir=/var/run/pluto/
-  virtual_private=%v4:10.0.0.0/8,%v4:192.168.0.0/16,%v4:172.16.0.0/12,%v4:!<LOCAL_SUBNET>
+  virtual-private=%v4:10.0.0.0/8,%v4:192.168.0.0/16,%v4:172.16.0.0/12,%v4:!<LT2P_CIDR>,%v4:!<XAUTH_CIDR>
```

Comment:
does the `-` vs `_` matter here?

Reply:
Docs definitely say "virtual-private" today. Wonder if it supported both in the past? Or if this just never worked?

https://libreswan.org/man/ipsec.conf.5.html